api/admin/api.go

```diff
@@ -63,6 +64,7 @@ func (a *Api) callApi(ctx context.Context, method string, path interface{}, requ
 		body,
 	)
 	if err != nil {
+		a.Config.ErrorLog(err)
```

Comment:
@RTLcoil,  The usage looks a bit weird, like accessing log through `Config`. Having this code all over the project would look a bit strange... Can we put it in some other common place?